client/scala/src/main/scala/ai/verta/repository/Commit.scala

```diff
@@ -282,4 +285,20 @@ class Commit(
     else
       Success(newCommit)
   }
+
+  /** Return ancestors, starting from this Commit until the root of the Repository
+   *  @return a list of ancestors
+   */
+  def log()(implicit ec: ExecutionContext): Try[List[Commit]] = {
```

Comment:
I recommend using a Stream to allow for pagination later.

Reply:
Fixed.